go.mod

```diff
@@ -120,3 +120,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
```

Comment:
what's the story with this replacement?

Reply:
Without it we get:
```
$ go mod tidy 
go: downloading vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787
github.com/openshift/cluster-kube-controller-manager-operator/cmd/cluster-kube-controller-manager-operator imports
	github.com/openshift/library-go/pkg/operator/staticpod/prune tested by
	github.com/openshift/library-go/pkg/operator/staticpod/prune.test imports
	vbom.ml/util/sortorder: unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp: lookup vbom.ml: no such host
```
This library is an dependency of `go-library`.

It looks like this library was [moved to github](https://github.com/kubernetes/kubectl/issues/925) some time ago, and I saw this replacement in few [Openshift components](https://github.com/search?q=org%3Aopenshift%20%22replace%20vbom.ml%2Futil%22&type=code).

I guess the vendoring hid the issue up until now?

Reply:
yeah, i guess so, thanks for the explanation!

Reply:
Should not it be pulled from the library-go go.mod file automatically?

Reply:
Running `go mod tidy` locally just changing api/library-go to `release-4.14` branch works without the need to set the replace.

Reply:
https://github.com/openshift/cluster-kube-controller-manager-operator/compare/master...ingvagabund:cluster-kube-controller-manager-operator:go-mod-tidy-experiment?expand=1

Reply:
hum... which version of go do you use?
```
$ go version
go version go1.20.4 linux/amd64
```

Reply:
```
$ go version
go version go1.20.2 linux/amd64
```

Reply:
The same works for me with 1.20.4 as well

Reply:
same it works for me with:
```
podman run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp golang:1.20.4 sh -c "go mod tidy && go mod vendor && go mod verify"
```

but the one I have on my system (fedora) keeps failing :thinking:

Reply:
I can reproduce with the fedora image:
```
podman run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp fedora sh -c "dnf install -y golang && go mod tidy && go mod vendor && go mod verify"
```

Reply:
Done!